### PR TITLE
FAQ: Send a patch: add of email address

### PR DESCRIPTION
**Pull request by mlep**

### CODE REVIEW

File: _faq/80_developers.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ A: We encourage you to read [Translating JabRef](https://github.com/JabRef/jabre
 ## Q: I want to send a patch. Where can I do that?
 
 A: We really prefer the pull request approach of GitHub.
-In case you want to contribute something, but git seems to be too hard for you, please get in touch with the developers to work out other ways of contributing.
+In case you want to contribute something, but git seems to be too hard for you, please get in touch with the developers to work out other ways of contributing (jabrefmail+webfeedback@gmail.com).
```
Comment (simonharrer) [inline _faq/80_developers.md]: I think we should remove this. We only want to work with git on github.

Comment (matthiasgeiger) [inline _faq/80_developers.md]: :+1: But a link to https://github.com/JabRef/jabref/blob/master/CONTRIBUTING.md would be good. 
Suggestion: `You can find more information on how to contribute [here at GitHub](https://github.com/JabRef/jabref/blob/master/CONTRIBUTING.md).`

Comment (mlep) [inline _faq/80_developers.md]: Note that we already have:
>Q: I am a programmer. How can I join the JabRef project?
>
>A: We follow the usual GitHub pull request approach. We have a lot of open issues. If you want to be sure to work on something the core developers currently do not handle, check the issues tagged with asking-for-a-pull-request. See our CONTRIBUTING.md for detailed information and further links.

If we keep the question, shall we say patches are not accepted?

Comment (simonharrer) [inline _faq/80_developers.md]: No, just remove the question about the patch.

**Pull request merged**